sklearn/ensemble/iforest.py

```diff
@@ -144,6 +144,24 @@ class IsolationForest(OutlierMixin, BaseBagging):
         is defined in such a way we obtain the expected number of outliers
         (samples with decision function < 0) in training.
 
+    Examples
+    --------
+    >>> import numpy as np
```

Comment:
A simpler example may be better?

```py
>>> import numpy as np
>>> from sklearn.ensemble import IsolationForest
>>> rng = np.random.RandomState(42)
>>> X = [[-1.1], [0.2], [101.1], [0.3]]
>>> clf = IsolationForest(random_state=0).fit(X)
>>> clf.predict([[0.1], [90], [0]])
array([ 1, -1,  1])
```

Reply:
Hi @thomasjpfan thanks for your feedback! I totally agree and already made the changes :)